packages/components/date-picker/src/use-date-picker-base.ts

```diff
@@ -234,10 +239,15 @@ export function useDatePickerBase<T extends DateValue>(originalProps: UseDatePic
     "data-invalid": dataAttr(originalProps?.isInvalid),
   } as DateInputProps;
 
+  const labelPlacement = useLabelPlacement({
+    labelPlacement: originalProps.labelPlacement,
+    label,
+  });
+
   const timeInputProps = {
     ...userTimeInputProps,
     size: "sm",
-    labelPlacement: "outside-left",
```

Comment:
@macci001 this should always be "outside-left"

Reply:
Done in: [1a0a128](https://github.com/nextui-org/nextui/commit/1a0a1281fef181234dae9c9993861b2e15e2971b)